RELEASE/scripts/autoscend/paths/zombie_slayer.ash

```diff
@@ -211,3 +211,15 @@ boolean zombieSlayer_acquireHP(int goal)
 	return my_hp() >= goal;
 }
 
+
+boolean zslayer_usable(familiar fam){
+	if (!in_zombieSlayer)
+	{
+		return true;
+	}
+	if (contains_text(fam.attributes, "undead"))
+	{
+		return true;
+	}
+	return false;
+}
```

Comment:
please make this function zombieSlayer_usable so it matches the naming convention of the other functions in this file (I made suggestions for all of these). also here's quick simplification of the function:
```suggestion
boolean zombieSlayer_usable(familiar fam)
{
	if (!in_zombieSlayer())
	{
		return true;
	}
	return contains_text(fam.attributes, "undead"));
}

```